lib/author.rb

```diff
@@ -0,0 +1,13 @@
+class Author
+  attr_reader :name, :books
+  def initialize(author_info)
+    @name = author_info[:first_name] + " " + author_info[:last_name]
```

Comment:
Great use of concatenation - you could also do something like `@name = "#{author_info[:first_name]} #{author_info[:last_name]}"`